Source/ButtonMerchant.swift

```diff
@@ -133,38 +133,16 @@ final public class ButtonMerchant: NSObject {
     @objc public static func handlePostInstallURL(_ completion: @escaping (URL?, Error?) -> Void) {
         core.handlePostInstallURL(completion)
     }
-    
-    /**
-     Button ML features configuration.
-     */
-    public static var features: Configurable {
-        return core.system
-    }
-    
+
     /**
-     Tracks an order.
+     Reports an order to Button.
 
-     This signal is used to power the Instant Rewards feature for Publishers to notify
-     their customers as quickly as possible that their order has been correctly tracked.
+     - Parameters:
+       - order: Your order object to be reported.
+       - completion: An optional completion block taking an optional error.
 
-     - Attention:
-     This does not replace server-side order reporting to Button.
-     [See: order reporting](https://developer.usebutton.com/guides/merchants/ios/report-orders-to-button#report-orders-to-buttons-order-api)
-     */
-    @available(*, deprecated, message: "Use ButtonMerchant.reportOrder(order:completion:) instead")
-    @objc public static func trackOrder(_ order: Order, completion: ((Error?) -> Void)? = nil) {
-        core.trackOrder(order, completion)
-    }
-    
-    /**
-     Reports an order.
-     
-     This signal is used to power the Instant Rewards feature for Publishers to notify
-     their customer as quickly as possible that their order has been correctly tracked.
-     
-     - Attention:
-     This does not replace server-side order reporting to Button.
-     [See: order reporting](https://developer.usebutton.com/guides/merchants/ios/report-orders-to-button#report-orders-to-buttons-order-api)
+     See also: Reporting Orders to Button
+     ([docs](https://developer.usebutton.com/guides/merchants/ios/report-orders-to-button#report-orders-to-buttons-order-api))
```

Comment:
The url for the report orders on Android and iOS is most likely going to change when @ecgreb moves it to a new page. 

See https://github.com/button/button-developers/pull/404#pullrequestreview-257635823 for more information

Reply:
@ecgreb will update the path to redirect to the right location as part of that PR.